ImperatorToCK3/Source/CK3Outputter/outTitles.cpp

```diff
@@ -8,7 +8,12 @@
 
 
 
-void CK3::outputTitleHistory(const std::shared_ptr<Title>& title, std::ofstream& outputStream) {
+using std::shared_ptr;
+using std::string;
+using std::map;
```

Comment:
Not set while you're at it?

Reply:
There's only one occurence, but I will add it for consistency.